kmod/igb/Makefile

```diff
@@ -41,6 +41,11 @@ endif
 # PTP is required for AVB support
 CFLAGS_EXTRA += -DIGB_PTP
 
+# Add 32 bit ioctl support
+ifeq ($(32BIT_IOCTL_SUPPORT),y)
+CFLAGS_EXTRA += -DSUPPORT_32BIT_IOCTL
+endif
+
```

Comment:
This change seems to me to be the main one relevant to the title of this pull request. I think it should be a separate commit.